addons/web/static/src/views/fields/properties/property_value.js

```diff
@@ -198,9 +199,18 @@ export class PropertyValue extends Component {
         } else if (this.props.type === "date") {
             newValue = newValue && serializeDate(newValue);
         } else if (this.props.type === "integer") {
-            newValue = parseInt(newValue) || 0;
+            try {
+                newValue = parseInteger(newValue) || 0;
+            }
+            catch {
```

Comment:
I think the format you used for "float" is the correct one?
(Still no linter? :eyes: )

Reply:
ah ineed, it should be inline with the `}`

but yes, my linter is configured, but it doesn't detect this error :/

which `.eslintrc` do you use ? (I have one I found on our github ~3 years ago, not sure it's still the correct one)

Reply:
edit: I found how to detect it, all good